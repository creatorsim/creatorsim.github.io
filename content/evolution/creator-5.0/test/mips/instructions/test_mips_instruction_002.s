
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:

    li $t0, 10
    addi $t1, $t0, -30 # 10 - 30
    addi $t2, $t1, 411 # x9 + 411
    addi $t3, $t2, 12  # x10 - 12
