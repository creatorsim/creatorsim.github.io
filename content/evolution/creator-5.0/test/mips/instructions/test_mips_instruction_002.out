
cr[PC]:0x10; ir[8,t0]:0xa; ir[9,t1]:0xffffffec; ir[10,t2]:0x187; ir[11,t3]:0x193; keyboard[0x0]:''; display[0x0]:''; 

